EphemeralElixir/EphemeralElixir
# Get new users stored to socket with latitude and longitude data

#### What does this PR do?
- Adds functionality that will allow new user to data to be stored in client along while updating all users activeUsers object
#### Where should the reviewer start?
- In socket.client.js and socket.server.js
  #### How should this be manually tested?
- If possible, open up a few tabs and login multiple times with facebook and check activeUsers object
  #### Any background context you want to provide?
  N/A
  #### What are the relevant tickets on Pivotal Tracker?
  N/A
  #### Provide screenshots (if appropriate).
  N/A
